js/ui/speed_dial_action/speed_dial_main_item.js

```diff
@@ -33,10 +33,9 @@ const SpeedDialMainItem = SpeedDialItem.inherit({
             hint: "",
             label: "",
             actions: [],
-            visible: true,
             activeStateEnabled: true,
             hoverStateEnabled: true,
-            indent: 56,
+            indent: 55,
```

Comment:
Why?

Reply:
By generic design, indent between the main button and action button should be 16px, but we have 17px now